.github/workflows/integration.yml

```diff
@@ -0,0 +1,84 @@
+name: Integration Tests
+
+on:
+  pull_request:
+    branches-ignore:
+      - 'ubuntu/**'
+
+concurrency:
+  group: 'ci-${{ github.workflow }}-${{ github.ref }}'
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: sh -ex {0}
+
+env:
+  RELEASE: bionic
+
+jobs:
+  package-build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v3.0.0
```

Comment:
Here (and below, also for other actions), is there a reason for not going with `@v3` instead of `v3.x.y`, relying on the fact that compatibility shouldn't be broken in a given major version?

The official docs (e.g. https://github.com/actions/checkout) only specify the major.

Reply:
The only reason was to keep it consistent with the rest of workflows, but taking into account that they are official GitHub actions, I agree it is better to pin only major versions. Updated, thanks for pointing out that.

I will align the other workflows to this approach in a follow-up PR.